### ISSUE TITLE
Only limited to 20 playlists?

### ISSUE THREAD

**Issue by adhikasigit** [jal278/spotify-playlist-export#11]
I don't know if it's a restriction of Spotify's API or from this app itself, an answer to that at least would be great!

**Comment by shape55** [comment]
Yes, same question here.  I'm seeing only 20 out of 440 playlists loading as active links which I can click.  Also, a "Make CSV" link does not appear on the main page, only if i click on an individual playlist.

**Comment by MattStarfield** [comment]
Same. I was able to rearrange the order of my playlists on Spotify to get the ones I was looking for to show up in the exporter app, so that's a temporary workaround.

**Comment by nsnyder** [comment]
According to [this](https://developer.spotify.com/documentation/web-api/reference/playlists/get-list-users-playlists/), you can get a maximum of 50 at a time, up to 100,000 playlists. But the default is 20.

**Comment by adhikasigit** [comment]
Thanks for the info! Maybe its time to write my own

**Comment by shape55** [comment]
FYI, if you guys haven't tried [Pavel Komarov's fork](https://github.com/pavelkomarov/exportify) of watsonbox's Exportify, he solved the rate-limiting error in 2019.  So "Export All" works well now.  I'm able to download nearly 500 playlists with a single button click.  It can be accessed at [exportify.net](https://exportify.net).
